onnxruntime/src/lib.rs

```diff
@@ -322,154 +322,6 @@ impl Into<sys::GraphOptimizationLevel> for GraphOptimizationLevel {
     }
 }
 
-// FIXME: Use https://docs.rs/bindgen/0.54.1/bindgen/struct.Builder.html#method.rustified_enum
```

Comment:
Moved as-is into `tensor.rs`. I figured with this + the new trait it was making `lib.rs` pretty fat, and I think `crate::tensor` is a reasonable home for these types. WDYT?